teams/wg-secure-code.toml

```diff
@@ -7,12 +7,18 @@ members = [
     "Shnatsel",
     "tarcieri",
     "alex",
+    "pinkforest",
```

Comment:
please also include @amousset

Reply:
I would have but @amousset first needs to add himself in - will fix after this

Reply:
opened #855

Reply:
```
validation error: person `amousset` is not a member of any team and has no permissions
```

it seems it's the other way round, but maybe you can add me in your PR directly.

Reply:
Yeah I'll add you in via this PR

Reply:
Done I think